sdk/windows_pipe_config.go

```diff
@@ -0,0 +1,13 @@
+package sdk
```

Comment:
I'd include this in `windows_listener.go` instead of placing it in it's own file. Unless it "has" to be here because of build tags.